sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -891,9 +890,9 @@ def upload_manifest(
             digest = response_headers['Docker-Content-Digest']
             if not _validate_digest(data, digest):
                 raise ValueError("The digest in the response does not match the digest of the uploaded manifest.")
-        except Exception:
+        except Exception as e:
             if repository is None or manifest is None:
```

Comment:
Keep the `None` check here as the default error message is not clear:
- when `repository=None`, will get `ValueError: No value for given attribute` 
- when `manifest=None`, will get `azure.core.exceptions.HttpResponseError: Operation returned an invalid status 'Bad Request'` - is unclear